certbot-apache/certbot_apache/parser.py

```diff
@@ -62,9 +67,12 @@ def __init__(self, aug, root, vhostroot, version=(2, 4)):
         # Set up rest of locations
         self.loc.update(self._set_locations())
 
-        # Must also attempt to parse virtual host root
-        self._parse_file(self.vhostroot + "/" +
-                         constants.os_constant("vhost_files"))
+        self.existing_paths = copy.deepcopy(self.parser_paths)
```

Comment:
One thing that worries me a bit is this is never updated if an `Include` directive is added later. What if we moved the `add_include` function into the parser and keep this value up to date?

Reply:
Nice catch! This functionality is now added to the PR